hawkular-alerts-engine/src/main/java/org/hawkular/alerts/engine/service/PartitionManager.java

```diff
@@ -72,6 +73,18 @@
     boolean isDistributed();
 
     /**
+     * Show additional information about partition status.
+     * In distributed scenarios
+     *  - getStatus().get("currentNode") will store a string with the identifier of the current node
+     *  - getStatus().get("members") will store a string with a list comma identifiers of the nodes of the topology
```

Comment:
certainly it's harmless.  IDE typically just grabs from the super, so it saves you from having to have them in both places.  Not a big deal, just a note.